contacts.py

```diff
@@ -245,6 +245,23 @@ def sync(self, data, args):
             ret['locations'] = locations
         return ret
 
+    # admin_config get
+    def admin_config(self, data, args):
+        ret = {
+            'directory': self.directory_root,
+            'testing': self.testing
+        }
+        return ret
+
+    # admin_status get
+    def admin_status(self, data, args):
+        ret = {
+            # TODO-36 DAN - how to get some meaningful stat from rtree
+            #'geo_points': self.rtree.count(),
+            'id_len': len(self.ids)
```

Comment:
we should then probably make len work correctly (this one will only give you a count of the number of unique first two bytes of ids)

Reply:
Oops - do you want to add methods on Contacts for both of these ?

Reply:
yes, will add to this branch before i merge